core/src/main/java/com/novoda/noplayer/player/ExoPlayerCapabilities.java

```diff
@@ -1,14 +1,21 @@
 package com.novoda.noplayer.player;
 
-import com.novoda.noplayer.drm.DrmHandler;
-import com.novoda.noplayer.drm.DownloadedModularDrm;
-import com.novoda.noplayer.drm.StreamingModularDrm;
+import com.novoda.noplayer.drm.DrmType;
+
+import java.util.Arrays;
+import java.util.List;
 
 class ExoPlayerCapabilities implements PlayerCapabilities {
 
+    private static final List<DrmType> SUPPORTED_DRM_TYPES = Arrays.asList(
```

Comment:
worth a `Collections.unmodifiableList()` (and on `AndroidMediaPlayerCapabilities` as well)?